app/src/main/AndroidManifest.xml

```diff
@@ -3,23 +3,30 @@
     package="com.example.weather">
```

Comment:
Em nên đổi tên cho chuyên nghiệp.
https://stackoverflow.com/questions/16804093/rename-package-in-android-studio
```suggestion
    package="vn.sunasterisk.weather">
```